bin/alluxio

```diff
@@ -79,31 +79,50 @@ EOF
 }
 
 function killAll {
-  if [[ $# -ne 1 ]]; then
-    echo "Usage: alluxio killAll <WORD>" >&2
+  if [[ $# -gt 2  ]]; then
+    echo "Usage: alluxio killAll [-s] <WORD>" >&2
```

Comment:
Did the `-s` only apply to AlluxioWorker?
In this usage, i will feel -s the soft kill option apply to all processes but looks like it only use in AlluxioWorker?